tests/test_handlers_multiyear.py

```diff
@@ -194,7 +202,7 @@ def test_cli(runner):
         LOGGERS.clear()
 
 
-@pytest.mark.parametrize(('dset', 'group'), [
+@ pytest.mark.parametrize(('dset', 'group'), [
```

Comment:
Wait, how is this not a syntax error? I see the tests passed but I'm baffled.... Did not know you can add a space between the "@" and the name of the decorator. In any case, is this intentional?

Reply:
Very surprising, no it is not intentional, will fix.